src/plugins/explore/public/application/utils/languages/ppl/default_prepare_ppl_query/default_prepare_ppl_query.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Query } from '../../../../../../../data/common';
+import { stripStatsFromQuery } from '../strip_stats_from_query';
+import { getQueryWithSource } from '../get_query_string_with_source';
+import { QueryWithQueryAsString } from '../../types';
+
+/**
+ * Default PPL query preparation for tabs (removes stats pipe for histogram compatibility)
+ */
+export const defaultPreparePplQuery = (query: Query): QueryWithQueryAsString => {
```

Comment:
tbh i wouldnt know that all defaultPreparePplQuery only strips the stats call. are we sure this is needed?

Reply:
it kind of conflicts with the initial query

Reply:
this is very messy and its from previous naming. I also agree we should revise these. The comment makes it imply it is just for histogram, maybe the name should say that more explicitely. @ananzh ?